docs/OTAnalytics/usage-ui.md

```diff
@@ -0,0 +1,434 @@
+# Usage UI
```

Comment:
Since there is no other type of usage described, I would go with "Usage" instead of "Usage UI"

Reply:
Usage CLI is the next chapter. Therefore, we stick to "Usage UI"

Reply:
Ok - maybe "GUI" or better "Graphical User Interface"? Don´t know if all of our potential users know what a "ui" or "gui" is...